src/context/Theme.tsx

```diff
@@ -105,6 +109,10 @@ const themeSettings = createTheme({
                         right: 0,
                         top: 0,
                         width: '100%',
+                        zIndex: buttonHoverIndex,
+                    },
+                    '& 	.MuiChip-deleteIcon': {
+                        zIndex: chipDeleteIndex,
```

Comment:
The custom styling above is what was causing the chips to not be able to be deleted. So making sure the delete icon is higher.